docs/admin/general-upgrade-guidelines.rst

```diff
@@ -0,0 +1,73 @@
+.. highlight:: sh
+
+.. _general_upgrade_guidelines:
+
+==========================
+General upgrade guidelines
+==========================
+
+Before kicking off an upgrade, there is a set of best practices to ensure the best outcome. Below you may find the five fundamental steps to prepare for an upgrade.
+
+.. NOTE::
+
+   This is not an exhaustive list, so you should consider your organization's specific needs and incorporate any additional steps or considerations that are relevant to your environment.
+
+Upgrade planning
+================
+
+Acknowledge breaking changes
+-----------------------------
+
+Review the `release notes`_ and documentation for the target version to understand any potential impact on existing functionality. 
+In case of a minor version upgrade, you should consider the intermediate versions. For example, when upgrading from 5.1 to 5.3, besides reviewing 5.3 release notes, the recommendation is to check for version 5.2 as well.
```

Comment:
> In case of a minor version upgrade, you should consider the intermediate versions

That also applies to major version upgrades (e.g. from 4.x to 5.y), the minor versions in between also need to be checked.

> the recommendation is to check for version 5.2 as well

I would phrase that more strongly, it's not just recommended, it needs to be done. Otherwise, breaking changes will be missed.

Reply:
If I do an upgrade I feel more confident checking all the release notes, but I think they have it as a policy to only do breaking changes on major releases.